Formula/fx.rb

```diff
@@ -18,14 +16,14 @@ class Fx < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "91533eff62e86951a9b67f7d9ced7673627add44ac66320199f8336b6fc576ad"
   end
 
-  depends_on "node"
+  depends_on "go" => :build
```

Comment:
```suggestion
  depends_on "go" => :build
  depends_on "node"
```
Seems like this is still needed.

Reply:
I'm gonna fix fx to not rely on nodejs in simple cases.

Reply:
If it's needed at any point (e.g. not simple cases) then we should probably still keep the `node` dependency.

Reply:
No, for main purpose of fx (terminal JSON viewer) nothing extra needed. 

Reducer's language can be chosen by users: JS, Python or Ruby. We don't need to add nodejs or ruby as deps here.

Reply:
Should be fixed now with introduction builtin reducers.